src/tool_parsecfg.c

```diff
@@ -43,7 +43,7 @@
 
 static const char *unslashquote(const char *line, char *param);
 
-#define MAX_CONFIG_LINE_LENGTH (100*1024)
+#define MAX_CONFIG_LINE_LENGTH (10*1024*1024)
```

Comment:
Could we document that somewhere? 

https://github.com/curl/curl/blob/8d9c1a8aa84a57cb1abb1e5f5e9e82c12ca71e64/docs/cmdline-opts/config.d#L17-L18

Maybe we should add to the above sentence `within the limit of 10MiB per line`, wdyt?

Reply:
Fair!